### PR TITLE
LPS-191610 | Modify and Unignore testcase to make assertions over the correct sample file

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Modify and Unignore testcase to make assertions over the correct sample file

Jira ticket:
https://liferay.atlassian.net/browse/LPS-191610

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Review by yanan — CHANGES_REQUESTED** [review]
@Hijky could you also help check if the downloaded sample file is empty or not if there is no object entry? Also, could you help make sure with developers if the sample file without an actual object entry is expected or not? Then we can update our existing tests accordingly, thanks

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/objectEntry_import_sample.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+keywords,externalReferenceCode,taxonomyCategoryIds,name
```
Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/objectEntry_import_sample.csv:1]: What I am not sure if the csv file only contains fields not the data with an real object entry, please help check with developers, thanks.

Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/objectEntry_import_sample.csv:1]: What I am not sure if the csv file only contains fields not the data with a real object entry, please help check with developers, thanks.

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Review by yanan — APPROVED** [review]

**Comment by Hijky** [comment]
The failures are not related to PR, and some of the failures have testing tickets, and the others are flaky.

**Comment by Hijky** [comment]
ci:forward:force

**Pull request closed without merging**